cibyl/sources/elasticsearch/api.py

```diff
@@ -184,7 +230,7 @@ def __init__(self: object, search_key: str,
                     "exists": {
                         "field": search_key
                     }
-                }
+                },
```

Comment:
Unneeded, but it does not really matter.

Reply:
Thanks Jorge. Removed.